docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -77,18 +77,18 @@ Command argument and behavior breaking changes are listed below.
 
 ## Known issues
 
-- Regarding generic update arguments, the only supported operation is using `--set` on the root level of a Graph object. Using `--add`, `--remove` or `--set` on sub-levels currently doesn't work (due to the underlying infrastructure change). For unsupported scenarios, you may use `az rest` to directly call [Microsoft Graph API](/graph/api/overview). Examples can be found at https://github.com/Azure/azure-cli/issues/22580.
-- Microsoft Graph related commands like `az ad` and `az role` will fail in Azure Stack environments which don't have Microsoft Graph support. Please use Azure CLI 2.36.0 or earlier versions for Azure Stack environments.
+- Regarding generic update arguments, the only supported operation is using `--set` on the root level of a Graph object. When you use `--add`, `--remove` or `--set` on sublevels currently doesn't work (due to the underlying infrastructure change). For unsupported scenarios, you may use `az rest` to directly call [Microsoft Graph API](/graph/api/overview). Examples can be found at https://github.com/Azure/azure-cli/issues/22580.
```

Comment:
> When you use `--add`, `--remove` or `--set` on sublevels currently doesn't work (due to the underlying infrastructure change).

This sentence is grammatically wrong.

Reply:
Please see PR [3381](https://github.com/MicrosoftDocs/azure-docs-cli/pull/3881) for correction.